types/spreedly-iframe-browser/index.d.ts

```diff
@@ -44,6 +44,10 @@ declare namespace spreedly {
     interface InitOptions {
         numberEl: string;
         cvvEl: string;
+        nonce?: string;
+        certificateToken?: string;
+        signature?: string;
+        timestamp?: string;
```

Comment:
Based on Spreedly's [documentation](https://developer.spreedly.com/docs/iframe-api-lifecycle#security-requirements), it seems like this options are now required.

```suggestion
        nonce: string;
        certificateToken: string;
        signature: string;
        timestamp?: string;
```